ddp-workspace/projects/ddp-dsm-ui/src/app/services/stackdriver-error-reporter.service.spec.ts

```diff
@@ -1,15 +0,0 @@
-// import { TestBed, inject } from '@angular/core/testing';
-
-import { StackdriverErrorReporterService } from './stackdriver-error-reporter.service';
-
-describe('StackdriverErrorReporterService', () => {
-  // beforeEach(() => {
-  //   TestBed.configureTestingModule({
-  //     providers: [StackdriverErrorReporterService]
-  //   });
-  // });
-  //
-  // it('should be created', inject([StackdriverErrorReporterService], (service: StackdriverErrorReporterService) => {
-  //   expect(service).toBeTruthy();
-  // }));
-});
```

Comment:
This file was not compiling. Since it is all commented stuff, just easier to get rid of it.